packages/cli/src/tools/npm.ts

```diff
@@ -29,7 +29,21 @@ export function isProjectUsingNpm(cwd: string) {
   return findUp.sync('package-lock.json', {cwd});
 }
 
-const registry = getNpmRegistryUrl();
+export const getNpmRegistryUrl = (() => {
+  // Lazily resolve npm registry url since it is only needed when initializing a
+  // new project.
+  let registryUrl = '';
+  return () => {
+    if (!registryUrl) {
+      try {
+        registryUrl = execSync('npm config get registry').toString().trim();
```

Comment:
```suggestion
        registryUrl = execSync('npm config get registry --workspaces=false --include-workspace-root').toString().trim();
```

as discussed on Discord.